user_student/templates/audio.html

```diff
@@ -1,56 +1,63 @@
 {% load student_tag %}
 
-<div style="margin:10px;padding:10px">
-    <p style="font-size:200%">
-        <b>Listen to Audio and Answer Below Questions</b>
-    </p>
-
-    <audio controls>
-         <source src="/media/{{request.session.questions|index_av:i}}" type="audio/mpeg">
-            Your browser does not support the audio element.
-    </audio>
-
-    <p style="font-size:200%">
-        <b>Multiple Choice Question</b>
-    </p>
-    {% for j in r %}
-
-        <form action="" onSubmit="return check(this)" id="{{j}}">
-          {% with  iteratedContent=request.session.question_content|getObject:j %}
-              <p></p>
-      
-              <p style="font-size:150%">{{iteratedContent.question}}</p>
-              <ol type="A">
-                  {% for option in iteratedContent.options %}
-                      <li>  
-                          <input type="radio" name="options" value="{{ option.option_description }}">{{ option.option_description }}</input>
-                      </li>
-                  {% endfor %}
-
-
-              </ol>
-              <input type="text" name="ans" value="{{ iteratedContent.answer }}" hidden>
-              <input type="text" name="c" value="0" hidden>
-              <input type="submit" name="sub" class="button" value="Check">
-           {% endwith %}
-          </form>
-
-    {% endfor %}
-    <br>
-    <br>
-    <br>
-
-    <input type="submit" id="submit" class="button" value="Submit" disabled style="background-color:darkgray;color:white">
+<div style="marting:10px;padding:10px">
```

Comment:
The spelling of margin is wrong. This error was fixed with the code clean up, together with removing the js reference to jquery from here. Please clarify why errors that were corrected have been re introduced with a blind copy and paste